### PR TITLE
Package Openssh: Add kerberos 5 support

### PR DESCRIPTION
**Pull request by dexter2cz**

### CODE REVIEW

File: packages/openssh/build.sh [hunk 1]
```diff
@@ -28,6 +28,7 @@ TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
 --with-pid-dir=$TERMUX_PREFIX/var/run
 --with-privsep-path=$TERMUX_PREFIX/var/empty
 --with-xauth=$TERMUX_PREFIX/bin/xauth
+--with-kerberos5=$TERMUX_PREFIX/usr
 ac_cv_func_endgrent=yes
```
Comment (tomty89) [inline packages/openssh/build.sh]: `$TERMUX_PREFIX`already includes `/usr`

Comment (dexter2cz) [reply packages/openssh/build.sh]: Sorry, my mistake. I tested it and it worked, so I didn't notice.
Fixed.

**Comment by aero** [comment]
openssh package is being updated from time to time like 
https://github.com/termux/termux-packages/commit/e696010d8907854dd194311ce8a02da0ca77a121
cc @fornwall 

Why this krb5-enabling patch is not merged even though there are krb5 pacakges already exists ?

**Comment by kpedro88** [comment]
@fornwall is it possible to merge this? it would be quite useful

**Comment by fornwall** [comment]
Thanks everyone! The updated openssh package at version `7.9p1-2` now contains kerberos support.

Update with `pkg up` as normal.

**Pull request merged**